supply-chain/audits.toml

```diff
@@ -12,6 +17,12 @@ who = "Kunal Mehta <legoktm@debian.org>"
 criteria = "safe-to-run"
 delta = "1.1.0 -> 1.1.2"
 
+[[audits.cc]]
```

Comment:
This review and find-msvc-tools below was straightforward, but I only had to do it in the first place because of https://github.com/mozilla/cargo-vet/issues/663.